datadog/resource_datadog_synthetics_test_.go

```diff
@@ -1411,7 +1411,7 @@ func syntheticsBrowserStepParams() schema.Schema {
 					Optional:    true,
 					Elem: &schema.Schema{
 						Type:         schema.TypeString,
-						ValidateFunc: validation.StringInSlice([]string{"Alt", "Control", "meta", "Shift"}, false),
+						ValidateFunc: validation.StringInSlice([]string{"Alt", "Control", "Meta", "Shift"}, false),
```

Comment:
I'm not familiar with this code base, but shouldn't there be a test updated/added?

Reply:
Good point, included modifiers in one of our test cases to validate "Meta" is now accepted.